domain/src/test/java/org/oppia/android/domain/classify/rules/textinput/TextInputEqualsRuleClassifierProviderTest.kt

```diff
@@ -23,18 +23,42 @@ import kotlin.test.fail
 @LooperMode(LooperMode.Mode.PAUSED)
 @Config(manifest = Config.NONE)
 class TextInputEqualsRuleClassifierProviderTest {
-  private val STRING_VALUE_TEST_UPPERCASE = createString(value = "TEST")
-  private val STRING_VALUE_TEST_LOWERCASE = createString(value = "test")
-  private val STRING_VALUE = createString(value = "string")
-  private val STRING_VALUE_TEST_EXTRA_SPACES = createString(value = "test  a  lot  ")
-  private val STRING_VALUE_TEST_SINGLE_SPACES = createString(value = "test a lot")
-  private val STRING_VALUE_TEST_NO_SPACES = createString(value = "testalot")
-  private val NON_NEGATIVE_VALUE = createNonNegativeInt(value = 1)
 
   @Inject
   internal lateinit var textInputEqualsRuleClassifierProvider:
     TextInputEqualsRuleClassifierProvider
 
+  @Inject
+  internal lateinit var interactionObjectTestBuilder: InteractionObjectTestBuilder
+
+  private val STRING_VALUE_TEST_UPPERCASE by lazy {
```

Comment:
As disucssed earlier, we should keep constants at the top, then the inject values, then other variables, followed by setUp function.

Reply:
But I am not sure because now you are using lazy because of which it looks nice to be kept above other variables. 

@Sarthak2601 @BenHenning Any opinion.

Reply:
Hmm. I'm not sure I have a strong opinion here, actually, or see specific value in changing this. Out of curiosity, why use ``by lazy``? Generally, we use that to avoid creating expensive objects or having side effects that should happen later. Neither cases seem relevant to constants, especially in a test.

Reply:
Ah, I just realized the ``by lazy`` is needed since it's relying on an injected value. That sort of makes them non-constants now. I think we should either:
1. Treat them as non-constants and rename them accordingly
2. Make ``InterationObjectTestBuilder`` a non-injected class and have companion functions to assemble constants in which case ``by lazy`` won't be needed anymore

Reply:
@anandwana001 I am in favour of second option.